dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -151,7 +147,7 @@ const TableWithFavorite = () => {
               redirect="auth"
             />
           )}
-          <div className="table-container" variant={PageSectionVariants.light}>
```

Comment:
This should probably be pulled out into a separate Jira/PR because it's breaking other PR tests as well. Do you know why this has suddenly become a problem?